### PR TITLE
docs: 📝 update title and add license badge

### PR DESCRIPTION
**Pull request by signekb**
## Description

These changes are to align the title and subtitle in the `README` with what we have agreed on in #40 and implemented in #57 .

While I was at it, I added a licence badge as well.

## Reviewer Focus

This PR needs a quick review. 

## Checklist

- [X] Ran spell-check
- [X] Rendered website locally

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
-# Collaboration with Git and GitHub: A gentle introduction to a team-based collaborative workflow using Git and GitHub
+# Git for Collaborative Teams
+#### An introductory course on team-based collaboration with Git and GitHub
```
Comment (lwjohnst86) [inline README.md:2]:
```suggestion
# Git for Collaborative Teams: An introductory course on team-based collaboration with Git and GitHub
```

In general, it's common practice to put the full project title as the title header of the README. So it avoids your issue you mentioned :stuck_out_tongue:

**Review by lwjohnst86 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (signekb) [reply README.md:2]: Yeahhhh, I just don't like how it looks - it's too long and looks kind of messy to me 🥴

Comment (signekb) [reply README.md:2]: My inspiration was how the subheadings look in GitHub docs, e.g.: 

image

Comment (lwjohnst86) [reply README.md:2]: The docs probably use `title` and `subtitle`, with custom CSS classes. READMEs are pretty basic.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
...
+
+[![](https://img.shields.io/badge/License-CC%20BY%204.0-lightgrey.svg)](https://creativecommons.org/licenses/by/4.0/)
 
 ## Description
```
Comment (lwjohnst86) [inline README.md:7]:
```suggestion
# :cat2: Git for Collaborative Teams

[![](https://img.shields.io/badge/License-CC%20BY%204.0-lightgrey.svg)](https://creativecommons.org/licenses/by/4.0/)

## Description

This is an introductory course on team-based collaboration with Git and GitHub.
```

@signekb what about this then?

Comment (signekb) [reply README.md:7]: I'll allow it :grin:

**Pull request merged**